__tests__/server.test.js

```diff
@@ -0,0 +1,47 @@
+const format = require("pg-format")
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const request = require('supertest')
+const express= require('express');
+const app = require("../app/app.js");
+const jestSorted = require('jest-sorted');
```

Comment:
There's no need to require jest sorted, once installed, you have access to its matchers already 😄